calculation/gmhazard_calc/gmhazard_calc/directivity/directivity.py

```diff
@@ -157,40 +159,63 @@ def compute_fault_directivity(
         constants.EventType.from_rake(rake),
     )
 
-    # Creating the array to store all fdi values
-    fdi_array = np.zeros((n_hypo_data.nhypo, len(sites), len(periods)))
-    phired_array = np.zeros((n_hypo_data.nhypo, len(sites), len(periods)))
-    for index, planes in enumerate(hypo_planes):
-        # Gets the plane index of the hypocentre
-        plane_index = plane_ind[index]
+    if n_procs == 1:
+        fdi, phired = [], []
+        for ix, cur_planes in enumerate(hypo_planes):
+            cur_fdi, (cur_phi_red, cur_predictor_functions, cur_other) = _compute_directivity_effect(
```

Comment:
One nitpicking, as we don't use `cur_predictor_functions` and `cur_other`, can these be just underscores?